chef/cookbooks/neutron/recipes/common_agent.rb

```diff
@@ -108,7 +109,11 @@
   end
 end
 
-if ['openvswitch', 'cisco', 'vmware'].include? neutron[:neutron][:networking_plugin]
+
+# openvswitch installation and configuration
+if neutron[:neutron][:networking_plugin] == 'vmware' or
+  (neutron[:neutron][:networking_plugin] == 'ml2' and
+   neutron[:neutron][:ml2_mechanism_drivers].include?("openvswitch"))
```

Comment:
> The old test was mentioning cisco, this seems to be lost here?

When selecting cisco, openvswitch is mandatory. So checking for openvswitch is enough.